modules/standard/BigInteger.chpl

```diff
@@ -2384,14 +2390,50 @@ When ``n/d`` does not produce an integer, this method may produce incorrect resu
       }
     }
   }
+  /*
+    .. warning::
 
+       n and d are deprecated - please use numer and denom respectively
+  */
+  deprecated
+  "n and d are deprecated - please use numer and denom respectively"
   proc bigint.divexact(const ref n: bigint, d: integral) {
     this.divexact(n, new bigint(d));
   }
 
+  proc bigint.divexact(const ref numer: bigint, const ref denom: bigint) {
+    if _local {
```

Comment:
Instead of copying the code in the body of the deprecated version, I think we want to *move* the code and have the deprecated versions just call the new ones.  That will prevent any modifications from accidentally happening to only one version and potentially getting lost (because it was to the deprecated version and just got deleted).